extension/src/popup/components/hardwareConnect/LedgerConnect/index.tsx

```diff
@@ -47,35 +50,55 @@ const parseLedgerError = (err: any): string => {
   return LEDGER_ERROR.OTHER;
 };
 
-export const LedgerConnect = () => {
+export const LedgerConnect = ({ newBipPath }: { newBipPath?: string }) => {
   const dispatch: AppDispatch = useDispatch();
-  const [isConnecting, setIsConnecting] = useState(false);
```

Comment:
changed the names to "detecting" and "ledgerConnectSuccessful", I think that makes it a little less confusing instead of 2 "connect" var names